api/pom.xml

```diff
@@ -50,6 +50,11 @@
 		</dependency>
 				
 		<!-- End OpenMRS core -->
+		<dependency>
+		    <groupId>org.skyscreamer</groupId>
+		    <artifactId>jsonassert</artifactId>
```

Comment:
Are you sure you want to include this artifact in the compiled module?

Reply:
Good catch @dkayiwa , @kaweesi this should have scope set to test, can can you please move it to the root pom file and define the version in the properties section, the module uses the dependency management tag FYI